streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/controller/FlinkEnvController.java

```diff
@@ -53,12 +52,8 @@ public RestResponse check(FlinkEnv version) {
   }
 
   @PostMapping("create")
-  public RestResponse create(FlinkEnv version) {
-    try {
-      flinkEnvService.create(version);
-    } catch (Exception e) {
-      throw new ApiDetailException(e);
```

Comment:
ApiDetailException is a special exception. The front end will display the exception through an alert pop-up window. If it is removed, the exception message will not be popped up to notify the user.

Reply:
How to show this exception info to the front end needs further improvement

<img width="929" alt="image" src="https://github.com/apache/incubator-streampark/assets/13284744/ffe8ab38-eb92-4178-b9c7-6bf874440a95">